src/main/java/reposense/util/FileUtil.java

```diff
@@ -106,7 +122,7 @@ public static void unzip(ZipInputStream zipInput, String destinationFolder) {
     public static void copyTemplate(String outputPath, String reportName) {
         String templateLocation = outputPath + File.separator + reportName;
         InputStream is = RepoSense.class.getResourceAsStream(Constants.TEMPLATE_ZIP_ADDRESS);
-        FileUtil.unzip(new ZipInputStream(is), templateLocation);
+        FileUtil.unzip(is, templateLocation);
```

Comment:
Why not pass the path of template zip as a parameter? i.e. ```String sourcePath```
So that the ```unzip``` has similar contract as ```zipJson```